orchestration/dagster_orchestration/hca_manage/verify_release_manifest.py

```diff
@@ -1,72 +1,96 @@
 """
 Given a file containing a list of constituent staging dirs for a DCP release (aka a manifest),
 verify that data has been loaded from each of them to the target DCP dataset and the count of loaded files
-matches the # in the staging area
+matches the # in the staging area.
 
-All datarepo load totals are inferred from a supplied start date for the import process.
+Files are determined to be loaded if they exist at the desired target path and crc as defined in the staging
+areas descriptors. It's possible that an expected file was loaded by another staging dir (i.e,. they both
+contain the same file). While this is discouraged, it's technically possible and we need to accommodate that.
+So, we check if the target path was loaded, disregarding the source staging dir.
 
 Example invocation:
-python verify_release_manifest.py -s 2021-03-24 -f testing.csv -g fake-gs-project -b fake-bq-project -d fake-dataset
+python verify_release_manifest.py -f testing.csv -g fake-gs-project -b fake-bq-project -d fake-dataset
 """
 import argparse
+import json
 import logging
 import sys
+from dataclasses import dataclass
+from collections import defaultdict
+from functools import partial
+from multiprocessing import Pool
 from urllib.parse import urlparse
 
 from google.cloud import bigquery, storage
-from google.cloud.bigquery.table import Row
 from google.cloud.storage.client import Client
 from dagster_utils.contrib.google import get_credentials
 
 logging.basicConfig(level=logging.INFO, format='%(message)s')
 
 
-def get_expected_load_totals(storage_client: Client, staging_areas: list[str]) -> dict[str, int]:
+@dataclass(frozen=True)
+class PathWithCrc:
+    path: str
+    crc: str
```

Comment:
do we perhaps want to really explicitly call out that this is the crc32c, given the confusion we've experienced before?